recipes/imagl/all/conanfile.py

```diff
@@ -117,10 +133,31 @@ def requirements(self):
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
+
+        # CMake generator must be set to Ninja when using Visual Studio to let choose the right 
+        # MSVC compiler version when multiple versions are installed on build machine. The problem
```

Comment:
The lingo I am most familiar with is "MSVC toolsets" and yes CMake which takes "VS2019" as an arg _always_ chooses the highest one installed.

Reply:
In my opinion, "MSVC toolset" is more often understood as the main version of toolset (140 for VS2013, 141 for VS2017 and 142 for VS2019). This is why I wanted to be explicit about the version of the compiler (the "142" toolset includes the 14.2x versions of the toolset, each version of which contains the MSVC compiler in version 19.2x ...).